src/components/Nav.svelte

```diff
@@ -1,5 +1,20 @@
 <script>
+  import { state } from "../auth.js";
+  import { onMount } from "svelte";
+  var value = [];
+  var name;
+  var redirectUrl;
   export let segment;
+
+  onMount(async () => {
+    var ref = window.location.origin;
+    redirectUrl = ref;
```

Comment:
You don't need a `ref` var here.

```suggestion
    redirectUrl = window.location.origin;
```